### ISSUE TITLE
Need a downvote?

### ISSUE THREAD

**Issue by rejon** [QiHardware/scale-telegram-bot#14]
Well we only allow upvoting, but what if someone already upvoted? SHould they be allowed to remove that vote? That would be a downvote, or demote

- [ ]

**Comment by rejon** [comment]
Another idea if we dont't want downvote, is one user, one vote, so if you click again, the vote is removed, better thana downvote. Better, right?

**Comment by clemsos** [comment]
@rejon Are we recreating reddit? Painful

**Comment by rejon** [comment]
@clemsos No, just need if a user already voted, remove the vote....and should be scale up, scale down
